examples/troopjs_require/readme.md

```diff
@@ -1,16 +1,21 @@
-# TroopJS TodoMVC Example
+# TroopJS • [TodoMVC](http://todomvc.com)
```

Comment:
Could you please keep unrelated changes (like the ones in this file) in separated commits? That helps us reviewing and keeps the commit log clean.